CMSgov/eAPD
# Tbolt/4047 Stop subform submission when hitting enter

Resolves #4047 

### Description
Updates subforms so they no longer submit/save the subform when pressing enter

### Signifcant changes or possible side effects
Introduces an additional accessibility concern. Noted in [#4162](https://app.zenhub.com/workspaces/cms-eapd-5a3823ad3659e94d8ee569be/issues/cmsgov/eapd/4162)

### Automated test cases written
Snapshots updated to include newly added markup

### Steps to manually verify this change

1. Navigate to each subform
2. Create a new one or edit and existing one
3. Fill out a field, press the enter key, verify it does not save

### This pull request is ready to review when


### This pull request can be merged when
